### DISCUSSION TITLE
听说法厄同出事了尊嘟假嘟

### DISCUSSION THREAD

**Discussion by 070612wwz** [share121/inter-knot#4906] [绝区零]
?

**Comment by bushigemengsishuihaiyinga** [comment]
这个委托我不接！

**Comment by shidijun** [comment]
不太清楚 据说法厄同的账号被注销了 号主可能有自己的原因吧 但是估计有小号的

**Comment by Minapocalis** [comment]
是法厄同的账号被盗了，真没想到，这太让我失望了。堂堂法厄同连这点防护措施都没有吗？不过，法厄同本人并没有被捉到。我相信，总有一天，他将重获荣光
![Uploading 1726151724827.jpg…]()

**Reply by hk0852** [reply]
你得先等待图片的uploading消失才能正常把图片发出来让别人看到，你现在的这张图片其他人是看不到的，发送失败了

**Comment by hk0852** [comment]
发生了什么

**Comment by tihuahuo** [comment]
这个委托我接了。

**Comment by cuijiaqi0406** [comment]
好像只要买几个加密录像母带作为报酬发一个委托就能勾引法厄同小号<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >
